README.md

```diff
@@ -47,5 +44,26 @@ To install Airflow in a developer playground setting (i.e. in our baremetal clus
 ./scripts/playground/build.sh -p $PASSWORD
 ```
 
-## Cleaning up the Playground
-To uninstall the stack, you can run `./scripts/playground/cleanup.sh`.
\ No newline at end of file
+### Cleaning up the Playground
+To uninstall the stack, you can run `./scripts/playground/cleanup.sh`.
+
+---
+
+## Installing the PerfScale Stack
```

Comment:
Can you define (in just 1 line ) what all is included in PerfScale Stack ?

Reply:
Yeah I'll add that change here https://github.com/whitleykeith/airflow-kubernetes/tree/ci